doc/en/announce/release-2.6.0.rst

```diff
@@ -73,7 +73,7 @@ holger krekel
 - cleanup setup.py a bit and specify supported versions. Thanks Jurko
   Gospodnetic for the PR.
 
-- change XPASS colour to yellow rather then red when tests are run
+- change XPASS colour to yellow rather than red when tests are run
```

Comment:
technically this is grammatical, but most spell checkers have been able to flag rudimentary grammar errors, so there are a handful of fixes for things like this...